content/en/blog/2023/sig-release-spotlight/sig-release-spotlight.md

```diff
@@ -0,0 +1,91 @@
+
+Everyone's way of getting involved within the Open Source community is different. SIG Release is a self-serving team, meaning that we write our own tools to be able to ship releases. We collaborate a lot with other SIGs, such as [SIG K8s Infra](https://github.com/kubernetes/community/blob/master/sig-k8s-infra/README.md), but all the tools that we used needs to be tailor-made for our massive technical needs, while reducing costs. This means that we are constantly looking for volunteers who’d like to help with different types of projects, beyond “just” cutting a release.
+
+Our current project requires a mix of skills like [Go](https://go.dev/) programming, understanding Kubernetes internals, Linux packaging, supply chain security, technical writing, and general open-source project maintenance. This skill set is always evolving as our project grows.
+
+For an ideal path, this is what we suggest:
+
+- Get yourself familiar with the code, including how features are managed, the release calendar, and the overall structure of the Release Team.
+- Join the Kubernetes community communication channels, such as [Slack](https://communityinviter.com/apps/kubernetes/community) (#sig-release), where we are particularly active. 
+- Join the [SIG Release weekly meetings](https://github.com/kubernetes/community/tree/master/sig-release#meetings)
+ which are open to all in the community. 
+ Participating in these meetings is a great way to learn about ongoing 
+ and future projects that you might find relevant for your skillset and interests.
+
+Remember, every experienced contributor was once in your shoes, and the community is often more than willing to guide and support newcomers. Don't hesitate to ask questions, engage in discussions, and take small steps to contribute.
+<figure>
+  <img src="/blog/2023/sig-release-spotlight/sig-release-meetings.png" alt="sig-release-questions">
+</figure>
+
+**5. What is the Release Shadow Program and how is it different from other shadow programs included in various other SIGs?**
+
+The Release Shadow Program offers a chance for interested individuals to shadow experienced members of the Release Team throughout a Kubernetes release cycle. This is a unique chance to see all the hard work that a Kubernetes release requires across sub-teams. A lot of people think that all we do is cut a release every three months, but that’s just the top of the iceberg.
+
+Our program typically aligns with a specific Kubernetes release cycle, which has a predictable timeline of approximately three months. While this program doesn’t involve writing new Kubernetes features, it still requires a high sense of responsibility since the Release Team is the last step between a new release and thousands of contributors, so it’s a great opportunity to learn a lot about modern software development cycles at an accelerated pace.
```

Comment:
```suggestion
Our program typically aligns with a specific Kubernetes release cycle, which has a predictable timeline of approximately three months. 
While this program doesn’t involve writing new Kubernetes features, it still requires a high sense of responsibility. 
Since, the Release Team is the last step between a new release and thousands of contributors, 
so it’s a great opportunity to learn a lot about modern software development cycles at an accelerated pace.
```